packages/ngtools/webpack/src/transformers/replace_resources.ts

```diff
@@ -200,9 +192,11 @@ function visitComponentMetadata(
           if (inlineStyleFileExtension) {
             const data = Buffer.from(node.text).toString('base64');
             const containingFile = node.getSourceFile().fileName;
-            url = `${containingFile}.${inlineStyleFileExtension}!=!${InlineAngularResourceLoaderPath}?data=${encodeURIComponent(
-              data,
-            )}!${containingFile}`;
+            url =
```

Comment:
Nit: Similar to before, a comment with an example would be really useful to help communicate what this is trying to do.